sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestProxyTestBase.java

```diff
@@ -289,8 +287,7 @@ protected void beforeTest() {
             }
 
             List<HttpPipelinePolicy> policies = new ArrayList<>();
-            policies.add(new TimeoutPolicy(Duration.ofMinutes(1)));
-            if (!interceptorManager.isLiveMode() && !testContextManager.doNotRecordTest()) {
+            if (interceptorManager.isRecordMode() && !testContextManager.doNotRecordTest()) {
```

Comment:
sanitizers are also needed in playback mode.

Reply:
This is a path that only is reached when the test mode is non-playback